spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/TorchSpec.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.torch
+import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
+
+class TorchSpec extends FlatSpec with BeforeAndAfter with Matchers {
+  def torchCheck(): Unit = {
```

Comment:
Oh I get it now, yes you're right! huh, so cancel/assume only produces the cancelled status (vs aborted) when called directly within the method. Ugh, that's unfortunate. Looks good.